apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -157,7 +164,16 @@ export class SendMessageChat extends SendMessageBase {
           isRetry: false,
         })
       );
+
+      return {
```

Comment:
This feels that it needs stricter typing and solid return signature but we can tackle it separately.

Reply:
do you mean this one specifically or all of them?